### ISSUE TITLE
Deploy image without manager

### ISSUE THREAD

**Issue by BenedictQ** [bitnami/bitnami-docker-tomcat#98]
### Name and Version

bitnami/tomcat:9.0

### What steps will reproduce the bug?

Run a docker container with the image without an ENV vars

### What is the expected behavior?

The base image runs without a manager and without the need to set the `TOMCAT_PASSWORD` or `ALLOW_EMPTY_PASSWORD`  environment variables.

There is an option to ensure that the manager is not deployed.

### What do you see instead?

As far as I can tell the manager is not running on my Tomcat server. However, an error is output on the console causing the container to stop:

>  "The TOMCAT_PASSWORD environment variable is empty or not set. Set the environment variable ALLOW_EMPTY_PASSWORD=yes to allow a blank password. This is only recommended for development environments."

I therefore don't want to use the `ALLOW_EMPTY_PASSWORD` ENV var in production, but I'm also not interested in setting the `TOMCAT_PASSWORD` var as I don't even want the manager to be deployed.

### Additional information

I am using this image to deploy a war to a Tomcat server as ROOT. That is the only thing I want deployed on the Tomcat server.

I am not too familiar with Tomcat, but to me the documentation for this image suggests that the manager is being deployed by the statement:

> By default, a management user named manager is created and is not assigned a password. Passing the TOMCAT_PASSWORD environment variable when running the image for the first time will set the password of this user to the value of TOMCAT_PASSWORD.

The option `TOMCAT_INSTALL_DEFAULT_WEBAPPS` is apparently true by default, but I can't actually find where the manager is deployed. Either way, I would expect that setting it to false should mean the manager isn't deployed. In this case you shouldn't need to provide a password.

In conclusion, please let me know:

1. How use this image to deploy a war to the ROOT without a manager
2. How to safely avoid setting `TOMCAT_PASSWORD` or `ALLOW_EMPTY_PASSWORD` vars

**Comment by miguelaeh** [comment]
Hi @BenedictQ ,
try setting `TOMCAT_ENABLE_AUTH` to `no`.

**Comment by BenedictQ** [comment]
That works, thank you!

Is it worth adding this to the README? Firstly I note that the ALLOW_EMPTY_PASSWORD isn't documented alongside the others in the Environment Variables section. Secondly the interplay between the auth ones isn't clear - for example adding a little bit saying that if you set TOMCAT_ENABLE_AUTH then you don't need to set the TOMCAT_PASSWORD ?

**Comment by miguelaeh** [comment]
It seems that all the env vars are documented here: https://github.com/bitnami/bitnami-docker-tomcat#environment-variables

**Comment by BenedictQ** [comment]
ALLOW_EMPTY_PASSWORD  is not on that page. In addition, as I stated above, there's no explanation of how the variables relate to each other.

**Comment by miguelaeh** [comment]
Hi @BenedictQ ,
`ALLOW_EMPTY_PASSWORD` is not a valid variable. Check the envs [file](https://github.com/bitnami/bitnami-docker-tomcat/blob/master/9.0/debian-10/rootfs/opt/bitnami/scripts/tomcat-env.sh).
